index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
Stupid request but that way we're sure we won't forget 😅 
(Although I'm not sure whether it has any incidence on the built app)
```suggestion
    <title>Benchttp</title>
```